geometry_msgs/PoseStamped pose
bool freeRunStarted
bool freeRunByVirtualBumper
bool stopRunByVirtualBumper
